Marlin/planner.cpp

```diff
@@ -440,12 +440,17 @@ void check_axes_activity() {
         } else {
           fan_kick_end = 0;
         }
-    #endif//FAN_KICKSTART_TIME
+    #endif //FAN_KICKSTART_TIME
+    #ifdef FAN_MIN_PWM
+      #define CALC_FAN_SPEED ((tail_fan_speed == 0) ? 0 : ( (tail_fan_speed < FAN_MIN_PWM) ? FAN_MIN_PWM : tail_fan_speed ) )
```

Comment:
I kind of like the idea of having "1" be the slowest fan speed, "255" the fastest, and then scale all other values proportionally. So, the calculation would then be:
```
tail_fan_speed ? FAN_MIN_PWM + tail_fan_speed * (255 - FAN_MIN_PWM) / 255 : 0;
// Examples:
//   80 +   1 * (255 - 80) / 255 =  80
//   80 + 128 * (255 - 80) / 255 = 167
//   80 + 255 * (255 - 80) / 255 = 255
```